test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/AwaitableProcess.cs

```diff
@@ -91,28 +101,37 @@ private void StartProcessingOutput(StreamReader streamReader)
         {
             _source = _source ?? new BufferBlock<string>();
             Interlocked.Increment(ref _reading);
-            Task.Run(() =>
+            Task.Run(async () =>
             {
-                string line;
-                while ((line = streamReader.ReadLine()) != null)
+                Task<string> line;
+                while ((line = await Task.WhenAny(_tcs.Task, streamReader.ReadLineAsync())).Result != null)
```

Comment:
Gotcha. I was trying to figure out how well I need to look at the code. I'll review it 👍